File: MODUL-6/python/PRAK605-2310817210007-RAYMONDHARIYONO.py
ordo = int(input(""))
matriksA = []
matriksB = []
matriksAXB = []
print("Matriks A")

for i in range(ordo):
    row = list(map(int, input().split()))[:ordo]
    matriksA.append(row)

print("Matriks B")

for j in range(ordo):
    row = list(map(int, input().split()))[:ordo]
    matriksB.append(row)

for _ in range(ordo):
    matriksAXB.append([0] * ordo)

for a in range(ordo):
    for b in range(ordo):
        for c in range(ordo):
            matriksAXB[a][b] += matriksA[a][c] * matriksB[c][b]

print("Matriks AXB")

for p in range(ordo):
    for q in range(ordo):
        print(matriksAXB[p][q], end=" ")
    print()

File: MODUL-6/c/PRAK605-2310817210007-RAYMONDHARIYONO.c
#include <stdio.h>

void main(){
    int ordo,baris1,baris2;

    scanf("%d", &ordo);

    int arr1[ordo][ordo];
    int arr2[ordo][ordo];

    printf("Matriks A: \n");
    for(int i = 0; i < ordo; i++){
        for(int j = 0; j < ordo; j++){
            scanf("%d",&arr1[i][j]);
        }
    }

    printf("Matriks B: \n");
    for(int i = 0; i < ordo; i++){
        for(int j = 0; j < ordo; j++){
            scanf("%d",&arr2[i][j]);
        }
    } 
    
    int hasil[ordo][ordo];

    for(int i = 0; i < ordo; i++){
        for(int j = 0; j < ordo; j++){
            hasil[i][j] = 0;
            for(int h = 0; h < ordo; h++){
                hasil[i][j] += arr1[i][h] * arr2[h][j];
            }
        }
    }

    printf("Matriks AXB: \n");
    for(int i = 0; i < ordo; i++){
        for (int j = 0; j < ordo; j++){
            printf("%d ", hasil[i][j]);
        }
        printf("\n");
    }
}